tools/MatrixHelper/SettingsData.cs

```diff
@@ -51,7 +51,7 @@ public static Setting[] GetSettings()
             new("OTEL_EXPORTER_OTLP_LOGS_ENDPOINT", "Equivalent to `OTEL_EXPORTER_OTLP_ENDPOINT`, but applies only to logs.", string.Empty, "string", ExporterCategory),
             new("OTEL_EXPORTER_OTLP_METRICS_ENDPOINT", "Equivalent to `OTEL_EXPORTER_OTLP_ENDPOINT`, but applies only to metrics.", string.Empty, "string", ExporterCategory),
             new("OTEL_EXPORTER_OTLP_TRACES_ENDPOINT", "Equivalent to `OTEL_EXPORTER_OTLP_ENDPOINT`, but applies only to traces.", string.Empty, "string", ExporterCategory),
-            new("SPLUNK_REALM", "The name of your organization's realm, for example, `us0`. When you set the realm, telemetry is sent directly to the ingest endpoint of Splunk Observability Cloud, bypassing the Splunk Distribution of OpenTelemetry Collector.", string.Empty, "string", ExporterCategory),
+            new("SPLUNK_REALM", "The Splunk Observability Cloud realm where the telemetry should be sent. For example, `us0` or `us1`. Defaults to `none`, which means that data goes to an OpenTelemetry Collector deployed on localhost. When set to `none`, all telemetry data (traces, metrics, logs, profiles) is sent to the OpenTelemetry Collector. When set to any value other than none, only traces and metrics are sent directly to the specified Splunk Observability Cloud realm. Logs and profiling data are not sent.", string.Empty, "string", ExporterCategory),
```

Comment:
IMO it is not always true. When `none` it sends the data to the endpoint defined by env. vars. It is true only when no other evn vars are set.

Reply:
@Kielek is correct, realm set's the cloud endpoint only when endpoints are not specifically set using `OTEL_EXPORTER_OTLP_ENDPOINT`. Here seems to be bug like if OTEL_EXPORTER_OTLP_TRACES_ENDPOINT (etc) is used.

Reply:
Verified that logs still go to localhost while we use realm, which then causes failures in the log.
Possibly profiles also.

Should think how we fix it, if possible in the plugin or most probably add a note in the docs.